### ISSUE TITLE
tsconfig alias `~`

### ISSUE THREAD

**Issue by agamm** [t3-oss/t3-env#274]
When I followed the docs here:
https://env.t3.gg/docs/nextjs#use-your-schema

You've asked me to import `env` with `import { env } from "~/env";`

If I don't have a tsconfig alias for `~` I think it is relevant to say that the `@` would work if we place the `env` file/dir in the same directory of the `@` alias.

**Comment by juliusmarminge** [comment]
i don't think this is necessary and i can't think of any other docs that mention this either
